resource/cache/client.lua

```diff
@@ -51,6 +52,10 @@ CreateThread(function()
 
 		cache:set('weapon', hasWeapon and currentWeapon or false)
 
+		local playername = GetPlayerName(PlayerId())
```

Comment:
why don't you directly use the cached playerId instead of PlayerId() and use camelCase

Reply:
you're right, also just realized I don't have to cache the player name in a loop as it doesn't change, I can just cache it once after the script has loaded